aggregate/pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>tech.tablesaw</groupId>
     <artifactId>tablesaw-parent</artifactId>
-    <version>0.34.3-SNAPSHOT</version>
+    <version>0.35.1-SNAPSHOT</version>
```

Comment:
This file shouldn't be edited. I'm afraid you probably pulled while I was trying to cut a release earlier today, which failed and was rolled back

Reply:
ah  didn't realize it was rolled back.  I will unwind this tomorrow.

Reply:
Unfortunately Maven is a bit frustrating in this regard. It'll commit the version bump and push it to Github early in the release process. If it fails anywhere in the process it's already pushed the commit despite the release failing

Reply:
It looks like I got the release out (it takes an hour to show up, so I'll verify later). Unfortunately I had to do a force push so this PR is still showing a diff. Not sure the best way to solve. You might have to open a new PR?

Reply:
no problem. Thanks for getting it cleared-up.